sdk/create-dapp/package.json

```
@@ -0,0 +1,50 @@
+{
+	"name": "@mysten/create-dapp",
+	"author": "Mysten Labs <build@mystenlabs.com>",
+	"description": "A cli for creating new SUi dApps",
```

```suggestion
	"description": "A CLI for creating new Sui dApps",
```

casing on dApp here is different than above